tests/functional/aws-node-sdk/test/versioning/replicationBucket.js

```diff
@@ -0,0 +1,80 @@
+const assert = require('assert');
+const async = require('async');
+
+const withV4 = require('../support/withV4');
+const BucketUtility = require('../../lib/utility/bucket-util');
+
+const bucketName = `versioning-bucket-${Date.now()}`;
+
+
+function checkError(err, code) {
+    assert.notEqual(err, null, 'Expected failure but got success');
+    assert.strictEqual(err.code, code);
+}
+
+function checkNoError(err) {
+    assert.ifError(err, `Expected success, got error ${JSON.stringify(err)}`);
```

Comment:
This method is basically just a wrapper around ifError that stringifies the error, is that all you wanted to do? The default of ifError is to print the error anyway just not stringified I guess.

Reply:
I can't remember exactly when but it was to make the errors readable by a tool, `istanbul` may be? This is the reason you see this pattern across the test suite.

Reply:
Will resolve but maybe take a not of it somewhere else to come back to